scripts/socorro_aws_s3.sh

```diff
@@ -58,8 +57,6 @@ aws_secret_access_key = ${AWS_SECRET_ACCESS_KEY}
 EOF
 fi
 
-AWSOPTIONS="--endpoint-url=http://${AWS_HOST}:${AWS_PORT}/"
-
-echo "S3 container bucket is ${AWS_BUCKET}"
```

Comment:
I took this line out because it's confusing when you're making and removing buckets.